src/client/level-loader.worker.js

```diff
@@ -5,9 +5,11 @@ onmessage = function(message) {
 };
 
 let loadLevelData = function(url) {
+	postMessage({ event: "downloadStart" });
 	fetch(`${url}`)
 		.then((response) => {
 			if(response.ok) {
+				postMessage({ event: "downloadComplete" });
 				return response.arrayBuffer();
 			} else {
 				loadingFinished(false, `Unable to retrieve ${url} from server: ${response.status} - ${response.statusText}`);
```

Comment:
On Mac OSX 10.15.2, Safari 13.0.4 - running this branch locally failed to render the scene and was stuck 'Downloading...' the scene. This error was never caught and I was left with the notification displaying for an infinite amount of time which is something I wouldn't consider the best UX.

However, this issue might be very browser-specific or caused by other factors and might be expected. It's worth a note nonetheless.

Reply:
Since we haven't aimed to support IOS, OSX and Safari, I'm going to let this one slip by for now. If it still occurs in `dev` or `master` after merging, please make it an issue so it does not get forgotten.